public/docs/ts/latest/guide/i18n.jade

```diff
@@ -129,7 +129,7 @@ a(id="i18n-plural-pipe")
 
 .alert.is-important
   :marked
-    The ng-xi18n messages extractor doesn't support the plural pipe right now.
+    The ng-xi18n messages extractor doesn't support the plural pipe.
```

Comment:
If we want to say that we probably want to say that support for ICU messages in attributes will come soon.